src/test/e2e/api/admin/project/variants.e2e.test.ts

```diff
@@ -315,6 +413,11 @@ test('Invalid variant in PATCH also throws 400 exception', async () => {
     await db.stores.featureToggleStore.create('default', {
         name: featureName,
     });
+    await db.stores.featureEnvironmentStore.addEnvironmentToFeature(
+        featureName,
+        'default',
+        true,
+    );
```

Comment:
Because now features should be assigned to environments, we need to modify the test to account for this. I'm not fond of modifying tests to adapt to code changes made in production so I want to bring your attention as a reviewer to this part.

This is where we're fetching the feature environments https://github.com/Unleash/unleash/pull/2591/files#diff-24ad1ee0838798068127b66ed470a16e72f16cd368932059dbb6c6bd9f055b23R1256 where we were previously fetching just features: https://github.com/Unleash/unleash/pull/2591/files#diff-24ad1ee0838798068127b66ed470a16e72f16cd368932059dbb6c6bd9f055b23L1250